pkg/database/migrations/31_scenes_captions.up.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE `scene_captions` (
+  `scene_id` integer,
+  `language_code` varchar(255) NOT NULL,
+  `path` varchar(255) NOT NULL,
```

Comment:
Is path really necessary? Can't we infer the path from the language code and caption type?

Reply:
I suppose the path could be reconstructed with the language_code and caption type.